exporter/exporterwrapper/exporterwrapper.go

```diff
@@ -80,6 +82,12 @@ func PushOcProtoSpansToOCTraceExporter(ocExporter trace.Exporter, td data.TraceD
 	var errs []error
 	var goodSpans []*tracepb.Span
 	for _, span := range td.Spans {
+		log.Printf("Trace ID: %x%x\tSpan ID: %x\tParent Span ID: %x\tName: %s\n",
+			binary.LittleEndian.Uint64(span.TraceId[0:8]),
+			binary.LittleEndian.Uint64(span.TraceId[8:16]),
+			binary.LittleEndian.Uint64(span.SpanId),
+			binary.LittleEndian.Uint64(span.ParentSpanId),
```

Comment:
Print nil here probably if ParentSpanId is nil.

Reply:
Done.